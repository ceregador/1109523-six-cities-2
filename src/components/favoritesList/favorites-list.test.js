import React from 'react';
import FavoritesList from './favorites-list.jsx';
import Renderer from 'react-test-renderer';

it(`renders correctly`, () => {
  const tree = Renderer
    .create(
        <FavoritesList
          offers={[]}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
