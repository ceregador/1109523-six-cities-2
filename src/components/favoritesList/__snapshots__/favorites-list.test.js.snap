// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  className="page__main page__main--favorites"
>
  <div
    className="page__favorites-container container"
  >
    <section
      className="favorites"
    >
      <h1
        className="favorites__title"
      >
        Saved listing
      </h1>
      <ul
        className="favorites__list"
      />
    </section>
  </div>
</main>
`;
